desktop/core/src/desktop/lib/metrics/views.py

```diff
@@ -35,4 +35,4 @@ def index(request):
       'timestamp': datetime.datetime.utcnow().isoformat(),
       'metric': global_registry().dump_metrics(),
   }
-  return JsonResponse(rep, indent=indent)
```

Comment:
Here is the two different files:
[indent_is_not_none.before.txt](https://github.com/cloudera/hue/files/1179887/indent_is_not_none.before.txt)
[indent_is_not_none.after.txt](https://github.com/cloudera/hue/files/1179889/indent_is_not_none.after.txt)


The output format does not change, I modified here just because this place use indent in the old response;

Use indent just make json output pretty in this place;
I think there is no necessary to change the GET parameter;